src/javascript/app/pages/user/accounts.js

```diff
@@ -1,15 +1,17 @@
-const moment             = require('moment');
-const setIsForNewAccount = require('./account/settings/personal_details').setIsForNewAccount;
-const getCurrencies      = require('./get_currency').getCurrencies;
-const BinaryPjax         = require('../../base/binary_pjax');
-const Client             = require('../../base/client');
-const BinarySocket       = require('../../base/socket');
-const getCurrencyList    = require('../../common/currency').getCurrencyList;
-const FormManager        = require('../../common/form_manager');
-const getElementById     = require('../../../_common/common_functions').getElementById;
-const localize           = require('../../../_common/localize').localize;
-const State              = require('../../../_common/storage').State;
-const urlFor             = require('../../../_common/url').urlFor;
+const moment              = require('moment');
+const setIsForNewAccount  = require('./account/settings/personal_details').setIsForNewAccount;
+const getCurrencies       = require('./get_currency').getCurrencies;
+const BinaryPjax          = require('../../base/binary_pjax');
+const Client              = require('../../base/client');
+const BinarySocket        = require('../../base/socket');
+const getCurrencyFullName = require('../../common/currency').getCurrencyFullName;
+const getCurrencyNameList = require('../../common/currency').getCurrencyNameList;
+const getCurrencyList     = require('../../common/currency').getCurrencyList;
```

Comment:
wondering if it's better to import the whole currency module since we are using three of its functions